src/xrpld/app/misc/HashRouter.h

```diff
@@ -221,6 +235,7 @@ class HashRouter
         suppressionMap_;
 
     std::chrono::seconds const holdTime_;
+    std::chrono::seconds const relayTime_;
```

Comment:
The `HoldTime` has been 300s for [over 11 years](https://github.com/ximinez/rippled/commit/699ac3d25669f04f3b30974100df16d1cb287499), so it's a pretty safe bet that's a good value. However, I see your point about being able to tweak `RelayTime`.

I added a commit to add configuration options for these two values. I'm leaving them *undocumented* for now, because I don't want operators just randomly changing them for no particular reason.